ui/app/pages/confirm-add-token/confirm-add-token.component.js

```diff
@@ -103,10 +104,27 @@ export default class ConfirmAddToken extends Component {
               className="page-container__footer-button"
               onClick={() => {
                 addTokens(pendingTokens).then(() => {
+                  const pendingTokenValues = Object.values(pendingTokens)
+                  console.log(pendingTokenValues)
+                  pendingTokenValues?.forEach((pendingToken) => {
```

Comment:
Nit: This `?` isn't needed, because `Object.values()` always returns an array. Unless it throws, that is. I don't think it can throw in this case, since `pendingTokens` appears to always be defined (it defaults to an empty object, and is set to an empty object when cleared).

```suggestion
                  pendingTokenValues.forEach((pendingToken) => {
```